src/Website/google docs.json

```
@@ -0,0 +1,51 @@
+{
+    "google docs.editingDoc": {
+        "message": "Editing a document",
+        "description": "Displayed when the user is editing a Google Document"
+    },
+    "google docs.viewingDoc": {
+        "meessage": "Viewing a document",
+        "description": "Displayed when the user is viewing a Google Document"
+    },
+    "google docs.browsingDoc": {
+        "messages": "Browsing documents",
+        "description": "Displayed when the user is browsing their Google Documents"
+    },
+    "google docs.editingForm": {
+        "message": "Editing a form",
+        "description": "Displayed when the user is editing a Google Form"
+    },
+    "google docs.viewingForm": {
+        "message": "Viewing a form",
+        "description": "Displayed when the user is viewing a Google Form"
+    },
+    "google docs.browsingForm": {
+        "message": "Browsing Forms",
+        "description": "Displayed when the user is browsing their Google Forms"
+    },
+    "google docs.editingSheet": {
+        "message": "Editing a spreadsheet",
+        "description": "Displayed when the user is eiditng a Google Sheet"
+    },
+    "google docs.viewingSheet": {
+        "message": "Viewing a spreadsheet",
+        "description": "Displayed when the user is viewing a Google Sheet"
+    },
+    "google docs.browsingSheet": {
+        "message": "Browsing spreadsheets",
+        "description": "Displayed when the user is browsing their Google Sheets"
+    },
+    "google docs.editingSlide": {
+        "message": "Editing a slidesheet",
```

```suggestion
        "message": "Editing a presentation",
```
Use the name used by Google on its website (same for the other strings)